html/comparison.html.haml

```diff
@@ -15,6 +15,7 @@
     %style
       = Diffy::CSS
 
+    %script{:src => "/assets/machinery-base.js"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace